commit 7987b0332e6b660ac7992176daeede40cab98390
Author: Nils Philippsen <nils@redhat.com>
Date:   Tue Jun 16 17:02:49 2009 +0200

    patch: open-macro
    
    Squashed commit of the following:
    
    commit ab8fe801c4f82017988cb44cb79d82d286aa0de4
    Author: Nils Philippsen <nils@redhat.com>
    Date:   Tue Jun 9 17:57:45 2009 +0200
    
        don't inadvertently use glibc open() macro

diff --git a/backend/mustek_pp.c b/backend/mustek_pp.c
index 8c3f06a..7e9d094 100644
--- a/backend/mustek_pp.c
+++ b/backend/mustek_pp.c
@@ -1152,7 +1152,7 @@ sane_open (SANE_String_Const devicename, SANE_Handle * handle)
 
 	}
 	
-	if ((status = dev->func->open (dev->port, dev->caps, &fd)) != SANE_STATUS_GOOD) {
+	if ((status = (dev->func->open) (dev->port, dev->caps, &fd)) != SANE_STATUS_GOOD) {
 
 		DBG (1, "sane_open: could not open device (%s)\n",
 				sane_strstatus (status));
diff --git a/backend/pixma_common.c b/backend/pixma_common.c
index 2bcb3c1..c5e1e96 100644
--- a/backend/pixma_common.c
+++ b/backend/pixma_common.c
@@ -511,7 +511,7 @@ pixma_open (unsigned devnr, pixma_t ** handle)
   strncpy (s->id, pixma_get_device_id (devnr), sizeof (s->id) - 1);
   s->ops = s->cfg->ops;
   s->scanning = 0;
-  error = s->ops->open (s);
+  error = (s->ops->open) (s);
   if (error < 0)
     goto rollback;
   error = pixma_deactivate (s->io);
diff --git a/backend/plustek_pp.c b/backend/plustek_pp.c
index 13d1443..629e238 100644
--- a/backend/plustek_pp.c
+++ b/backend/plustek_pp.c
@@ -258,7 +258,7 @@ static int drvopen(	Plustek_Device *dev )
 
     DBG( _DBG_INFO, "drvopen()\n" );
 
-	handle = dev->open((const char*)dev->name, (void *)dev );
+	handle = (dev->open)((const char*)dev->name, (void *)dev );
 
 	tsecs = 0;
 
diff --git a/sanei/sanei_scsi.c b/sanei/sanei_scsi.c
index 69d5859..a594aba 100644
--- a/sanei/sanei_scsi.c
+++ b/sanei/sanei_scsi.c
@@ -5328,7 +5328,7 @@ sanei_scsi_find_devices (const char *findvendor, const char *findmodel,
     (*plugInInterface)->Release (plugInInterface);
     IOObjectRelease (scsiDevice);
 
-    ioReturnValue = (*scsiDeviceInterface)->open (scsiDeviceInterface);
+    ioReturnValue = ((*scsiDeviceInterface)->open) (scsiDeviceInterface);
     if (ioReturnValue != kIOReturnSuccess)
       {
 	DBG (5, "Error opening SCSI interface (0x%08x)\n", ioReturnValue);
